SimG4Core/PhysicsLists/plugins/module.cc

```diff
@@ -20,14 +22,19 @@
 #include "QGSPCMS_FTFP_BERT_EML95.hh"
 #include "QGSPCMS_FTFP_BERT_EML_New.hh"
 #include "QGSPCMS_FTFP_BERT_EML95msc93.hh"
+#include "QGSPCMS_FTFP_BERT_EMM.hh"
```

Comment:
I am wrong - they exist locally....so my local tests pass. Likely a correct procedure would be to check in another parallel area.